src/test/resources/AuthorConfigParserTest/authorconfig_unknownHeaders_test.csv

```diff
@@ -0,0 +1,3 @@
+Repository Location,Branch,Author's GitHub ID,Author's Emails,Author's Display Name,Author's Git Author Name,Ignore Glob
```

Comment:
Can we change the unknown headers to be something more obvious? Something like "Totally Not A Real Header" or something better. I can't really tell directly which header should not be there.